distribution/openrct2.d.ts

```diff
@@ -5362,7 +5366,6 @@ declare global {
         getObject(type: "footpath_addition", index: number): FootpathAdditionObject;
         getObject(type: "banner", index: number): BannerObject;
         getObject(type: "scenery_group", index: number): SceneryGroupObject;
-        getObject(type: "music", index: number): LoadedObject;
```

Comment:
This was removed, as it is unnecessary: it's caught by the `getObject(type: ObjectType, index: number): LoadedObject;` specification above.